godtools/App/Services/Renderer/Views/MobileContent/Views/ContentButton/MobileContentButtonViewModel.swift

```diff
@@ -113,23 +123,23 @@ class MobileContentButtonViewModel: NSObject, MobileContentButtonViewModelType {
         return 1
     }
     
-    var buttonType: MobileContentButtonType {
+    var buttonType: Button.Type_ {
         return buttonModel.type
     }
     
-    var buttonEvents: [MultiplatformEventId] {
+    var buttonEvents: [EventId] {
         return buttonModel.events
     }
     
     var buttonUrl: String {
-        return buttonModel.url ?? ""
+        return buttonModel.url?.absoluteString ?? ""
     }
     
-    var rendererState: MobileContentMultiplatformState {
-        return rendererPageModel.rendererState
+    var rendererState: State {
+        return renderedPageContext.rendererState
     }
     
     func buttonTapped() {
-        mobileContentAnalytics.trackEvents(events: buttonModel.getAnalyticsEvents(), rendererPageModel: rendererPageModel)
+        mobileContentAnalytics.trackEvents(events: buttonModel.analyticsEvents, renderedPageContext: renderedPageContext)
```

Comment:
you should use `buttonModel.getAnalyticsEvents(Trigger.CLICKED)` for fetching the analytics events to trigger here